### PR TITLE
Add latex

### PR DESCRIPTION
**Pull request by vyshnavsreeshan**

### CODE REVIEW

File: client/src/components/AddLatex/index.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    onChange = (selectedUsers) => {
+        this.setState({
+            selectedUsers: selectedUsers || [],
+        });
+        console.log(selectedUsers);
+    };
+
+    loadOptions = async (inputText, callback) => {
+        const response = await fetch(
+            `https://nalandaapi.herokuapp.com/api/newcustomer/search?searchString=${inputText}`
```
Comment (unnikrishnan-r) [inline client/src/components/AddLatex/index.js:31]: Can we move this to the API file and then use API.* to maintain consistency?

File: client/src/components/AddLatex/index.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+    render() {
+        return this.props.trigger ? (
+            <div className="newLatex">
+                {" "}
+                <div className="newLatexInner">
+                    <h1>Add Latex Collection</h1>
+                    <br></br>
+                    <AsyncSelect
+                        isMulti
```
Comment (unnikrishnan-r) [inline client/src/components/AddLatex/index.js:92]: do we need isMulti here?

File: client/src/pages/LatexCollection.js [hunk 3]
```diff
@@ -141,6 +157,10 @@ class LatexCollection extends Component {
                         columnDefs={this.state.columnDefs}
                         frameworkComponents={this.state.frameworkComponents}
                     ></AgGridReact>
```
Comment (unnikrishnan-r) [inline client/src/pages/LatexCollection.js:159]: What is the purpose of frameworkComponents?

**Review by unnikrishnan-r — CHANGES_REQUESTED** [review]
I have added review comments on corresponding pages

File: client/src/components/AddLatex/index.js (hunk 2 above)
Comment (vyshnavsreeshan) [reply client/src/components/AddLatex/index.js:92]: No sir even if we select multiple customers and add collection it reflects only for one.

Comment (vyshnavsreeshan) [reply client/src/components/AddLatex/index.js:92]: I will remove it and will make a new PR after making every other correction

File: client/src/pages/LatexCollection.js (hunk 3 above)
Comment (vyshnavsreeshan) [reply client/src/pages/LatexCollection.js:159]: It was a part of the code in Main Page forgot to remove when copied.

Comment (vyshnavsreeshan) [reply client/src/pages/LatexCollection.js:159]: Sir this is used to render the payment status.

**Review by unnikrishnan-r — APPROVED** [review]

**Pull request merged**